lighthouse-core/report/html/renderer/i18n.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
:tada:

Reply:
you're avoiding the condensed version though?? 😢 😢

Reply:
lol didnt notice, i just here to copy and paste stuff man